Documentation/gettingstarted/hubble-enable.rst

```diff
@@ -0,0 +1,41 @@
+Enable Hubble
+==============
+
+Hubble is a fully distributed networking and security observability platform
+for cloud native workloads. It is built on top of Cilium and eBPF to enable
+deep visibility into the communication and behavior of services as well as the
+networking infrastructure in a completely transparent manner.
+
+Deploy Hubble using Helm:
+
+.. parsed-literal::
+
+   helm upgrade cilium |CHART_RELEASE| \\
+      --namespace kube-system \\
```

Comment:
haven't figured out how to parameterize the namespace, for example gke uses `cilium` instead of `kube-system`.

Reply:
i found the same link :) it didn't work with sphinx unfortunately https://stackoverflow.com/questions/58930190/sphinx-and-restructuredtext-pass-parameter-in-include-directive#comment104245250_58930571

Reply:
added a step to specify the namespace as an env variable.